lib/ansible/plugins/inventory/netbox.py

```diff
@@ -50,10 +52,15 @@
         query_filters:
             description: List of parameters passed to the query string (Multiple values may be separated by commas)
             type: list
+            default: []
         timeout:
             description: Timeout for Netbox requests in seconds
             type: int
             default: 60
+        compose:
+            description: List of custom ansible host vars to create from the device object fetched from NetBox
+            default: {}
```

Comment:
Do we need `version_added` here?

Reply:
Netbox is not present in 2.7 so I guess that all of it will be present for 2.8